src/transformers/generation/logits_process.py

```diff
@@ -326,14 +329,18 @@ class RepetitionPenaltyLogitsProcessor(LogitsProcessor):
     ```
     """
 
-    def __init__(self, penalty: float):
+    def __init__(self, penalty: float, input_ids_seq_length: Optional[int] = 0):
```

Comment:
```suggestion
    def __init__(self, penalty: float, input_ids_seq_length: Optional[int] = None):
```
We prefer to default optional flags to `None` whenever possible and sensible. That way we can distinguish between no user input and user input = 0, which may be handy in the future 🤗